kmir/build_glr.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ "$1" == "llvm" ]; then
+    case $# in
+
+    1)
+        kmir init $(kbuild which llvm)
+        ;;
+    *)
+        poetry run kmir init $(poetry run kbuild which llvm)
+        ;;
+    esac
+fi
```

Comment:
I actually don't really want to use this script at all, but I couldn't get the conditional logic to work in the Makefile. Makefile stuff is not my strong suit.